pipeline/workflow/gene_config_all_genes.txt

```diff
@@ -0,0 +1,11 @@
+symbol,entrez_id,chr,end_hg38,ensembl_id,start_hg38,start_hg38_legacy_variants,end_hg38_legacy_variants,synonyms_ac_col,hgvs_cdna_default_ac,start_hg37,end_hg37,strand
```

Comment:
I ran a
```
 cat gene_config_*.txt  | grep -v -e '^symbol,' | sort | uniq
```
to see if for every gene there is only one line, that is, the information is consistent across all genes.
Turns out, that  `CHEK2` and `PALB2` in `gene_config_extended.txt` are different from e.g. in this file. I wonder which one it is we want. Also perhaps the `gene_config_extended.txt` may be obsolete now, so we could remove it.

Reply:
updated! i left all config files in place after updating content in case they prove useful later, can also remove if we don't need them